watertap/unit_models/osmotically_assisted_reverse_osmosis_base.py

```diff
@@ -268,8 +285,12 @@ def eq_recovery_mass_phase_comp(b, t, j):
         @self.Constraint(self.flowsheet().config.time, solute_set)
         def eq_rejection_phase_comp(b, t, j):
             return b.rejection_phase_comp[t, "Liq", j] == 1 - (
-                b.permeate_side.properties_out[t].conc_mass_phase_comp["Liq", j]
-                / b.feed_side.properties_in[t].conc_mass_phase_comp["Liq", j]
+                b.permeate_side.properties[t, b.first_element].conc_mass_phase_comp[
+                    "Liq", j
+                ]
+                / b.feed_side.properties[t, b.first_element].conc_mass_phase_comp[
+                    "Liq", j
+                ]
             )
```

Comment:
I just want to confirm do we still have `propterties_in` and `properties_out` for` permeate_side`? Not so clear why we want to use `propterties` instead.

Reply:
Yes -- this expression is equivalent to what its replacing in the 0D model. This is the "base" class so we really should be expressing these constraints in the common format for 1D and 0D.